### PR TITLE
OCPBUGS#3360:updates RHEL compute nodes support

### PR DESCRIPTION
**Pull request by JoeAldinger**
OCPBUGS-3360: updates RHEL compute nodes support for 4.12

Version(s):
4.12

Issue:
Post-installation configuration and Machine management [Jira](https://issues.redhat.com/browse/OCPBUGS-3360)
Release notes 4.9 #53211 
Jira [issue](https://issues.redhat.com/browse/OCPBUGS-4067) for RHEL compatibility for compute nodes 
[Jira](https://issues.redhat.com/browse/OSDOCS-4394)
https://issues.redhat.com/browse/OCPBUGS-2791

Link to docs preview:
[Post-installation configuration](https://52593--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/node-tasks.html) and [Machine management](https://52593--docspreview.netlify.app/openshift-enterprise/latest/machine_management/adding-rhel-compute.html)
Release notes 4.9 

QE review:
- [x] QE has approved this change.

Additional information:
This will need CM. 
@gpei PTAL for QE ack and comments
https://github.com/openshift/openshift-docs/pull/53211 4.9 RNs
#53648 4.10 RNs
https://github.com/openshift/openshift-docs/pull/53654 4.11 RNs

### CODE REVIEW

**Review by Wiharris — COMMENTED** [review]
/LGTM

**Comment by gpei** [comment]
LGTM

**Comment by kalexand-rh** [comment]
From a CM perspective, LGTM.

File: modules/rhel-compute-requirements.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ ifdef::openshift-origin[]
 ** Base OS: CentOS 7.4.
 endif::[]
 ifdef::openshift-enterprise,openshift-webscale[]
-** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[{op-system-base} 8.4 or 8.5] with "Minimal" installation option.
+** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[{op-system-base} 8.5 or 8.6] with "Minimal" installation option.
```
Comment (sdodson) [inline modules/rhel-compute-requirements.adoc:22]:
```suggestion
** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[{op-system-base} 8.5 or later] with "Minimal" installation option.
```

Comment (sdodson) [reply modules/rhel-compute-requirements.adoc:22]: For 4.11 this should be RHEL8 version 8.5 or later, for 4.10 this should be RHEL8 version 8.4 or later, and for 4.9 this should be RHEL7 version 7.9 or RHEL8 version 8.4 or later.

The goal is to convey that we set a minimum minor version and as long as it's greater than that but not RHEL9 we will support it.

Comment (sdodson) [reply modules/rhel-compute-requirements.adoc:22]: After discussing on Slack, this is more simply written as "8.6-8.7" which will be accurate for 4.12.

**Pull request merged**